ports/libmlservo/Cargo.toml

```diff
@@ -25,4 +25,4 @@ servo-egl = "0.2"
 simpleservo = { path = "../libsimpleservo/api", features = ["no_static_freetype"] }
 smallvec = { workspace = true }
 webxr = { git = "https://github.com/servo/webxr", features = ["ipc", "magicleap"] }
-webxr-api = { git = "https://github.com/servo/webxr", features = ["ipc"] }
+webxr-api = { git = "https://github.com/servo/webxr", features = ["ipc"] }
```

Comment:
The libmlservo directory should be removed, as well as this line: https://github.com/servo/servo/blob/master/Cargo.toml#L8